syntax = "proto2";

import "db_option.proto";

package proto.db;

message player_base {
	option (primary_key) = "id";
	optional sfixed32 id = 1;
	optional bytes name = 2;
}

message player_base_set {
	repeated player_base data_set = 1;
}